glusterd2/main.go

```diff
@@ -130,6 +132,13 @@ func main() {
 	super.ServeBackground()
 	super.Add(servers.New())
 
+	// Start dbus connection (optional for notifying firewalld)
+	if err := firewalld.Init(); err != nil {
+		log.WithError(err).Warn("firewalld.Init() failed")
+	}
+
+	pmap.Init()
```

Comment:
question:
do we need to do `pmap.Init()` even if firewall initialization fails?

Reply:
okay if initialization of firewalld fails I think even `firewalld.AddPort` won't work
what I was thinking if firewalld initialization fails, why want we set `notifyFirewalld` to `false` after pmap

Reply:
> think even firewalld.AddPort won't work what I was thinking

That is expected behaviour and is handled inside `firewalld.AddPort`.

> why want we set notifyFirewalld to false after pmap

`notifyFirewalld` is the config option, not a flag to decide to go ahead with dbus notification. There are separate guards for that. The local variable `isRunning` in firewalld package takes care of what you are looking for.